lib/ap/string/mempcpy.c

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2005-2014 Rich Felker, et al.
+ * Copyright (c) 2015-2016 Álvaro Jurado et al.
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE.mit file.
+ */
+
+#define _GNU_SOURCE
+#include <string.h>
+
+void *mempcpy(void *dest, const void *src, size_t n)
+{
+	return (void *)memcpy(dest, src, n) + n;
```

Comment:
man I'm an idiot. I apologize. I forgot that void * arithmetic is only valid in gcc, not real c11. You're going to kill me but you should change this back. Sorry :-(